app/views/layouts/partials/_navbar.erb

```diff
@@ -0,0 +1,20 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
```

Comment:
Maybe we should keep to one dialect and decide on ERB or Haml.

Reply:
I think that haml is good choice.

Reply:
Haml is great, but I think we should get other's input on that, since it has a completely different syntax. 😁 
@Sieron94PL @PrzemekPolec what do you think of haml? Are you okay with using it throughout project or would you prefer to stick to erb?